content/news/playtest-20191020.md

```diff
@@ -0,0 +1,43 @@
+A new playtest is now available for OpenRA's increasingly poorly named "Summer 2019" release. Thanks to everybody who tested [playtest-20190825](/news/playtest-20190825), this build should address all the issues and feedback that have been reported so far.
+
+The key changes in the playtest-20191020 include:
+
+* Fixed several issues with game loading/saving
+* Fixed several crashes and a desync issue
+* Fixed several glitches related to unit movement
+* Fixed several mission related issues in Red Alert and Tiberian Dawn
+* Fixed several compatibility issues with Linux and FreeBSD
+* Increased the default screen-edge scrolling speed
+* Refined the new spectator UI
+* Included further Red Alert balance adjustments
+
+We hope that this will be the last Playtest build before the next release, so head on over to our [download page](/download/), try it out, and let us know if you find any issues or have any other feedback before it is too late! As always, the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/XXXX) is available on GitHub if you would like to find out more.
+
+<hr>
+
+Season 8 of the Red Alert Global League is well underway. Resident RAGL wrangler netnazgul has been posting weekly wrap-up reports ([Week 1](https://forum.openra.net/viewtopic.php?f=85&t=21029), [Week 2](https://forum.openra.net/viewtopic.php?f=85&t=21031)) on the forum, covering the highlights and providing links to stream VODs for everybody who would like to keep up with the action.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20191020-ragl.gif" alt="Explosive RAGL action">
+
+RAGL Season 8 is packed with explosive action.
+</div>
+
+<hr>
+We have been relatively quiet about OpenRA's Tiberian Sun mod lately, but this does not mean it has been abandoned. Development has been focused mainly on resolving low-level game engine issues, which has been slow due to the complexity of the work involved and relative lack of manpower.
+
+Examples of this work can be seen in [release-20180923](/news/release-20180923/) which included renderer changes that roughly doubled TS's in-game performance, [release-20190314](news/release-20190314) which fixed a collection of game-breaking bugs with Nod's cloak generator technology, and the upcoming release will bring big improvements for deployable units and aircraft.
```

Comment:
`Eamples of this work can be seen in ... and the upcoming release will bring` should be `which will bring` I suppose.